### ISSUE TITLE
Error renewing certificate

### ISSUE THREAD

**Issue by tjmullicani** [hakwerk/labca#109]
I receive the following error when trying to renew the root or intermediate CA certificate.

![Capture](https://github.com/hakwerk/labca/assets/18095326/01556f9f-3d79-4e98-9d86-2afd5d9fd2ce)

```text
REPOSITORY                      TAG       IMAGE ID       CREATED        SIZE
ghcr.io/hakwerk/labca-boulder   latest    cab256ded9f5   8 days ago     336MB
ghcr.io/hakwerk/labca-control   latest    2f677e24246a   8 days ago     255MB
ghcr.io/hakwerk/labca-gui       latest    b6e9906d644a   8 days ago     215MB
mariadb                         10.5      72610f6b03ac   2 months ago   393MB
nginx                           1.25.3    a8758716bb6a   2 months ago   187MB
hashicorp/consul                1.15.4    66e415ee7888   6 months ago   172MB
```

```text
gui-1      | 2024/01/21 08:12:28 POST /manage
gui-1      | 2024/01/21 08:12:28 errorHandler: err=invalid semicolon separator in query
gui-1      | main._managePost({0x1397ff8, 0xc000ed81c0}, 0xc00017c900)
gui-1      |    /go/src/labca/main.go:1333 +0x17d
gui-1      | main.manageHandler({0x1397ff8, 0xc000ed81c0}, 0xc00017c900)
gui-1      |    /go/src/labca/main.go:1626 +0x6a
gui-1      | net/http.HandlerFunc.ServeHTTP(0xd2cc20?, {0x1397ff8?, 0xc000ed81c0?}, 0xc000ed81c0?)
gui-1      |    /usr/local/go/src/net/http/server.go:2136 +0x29
gui-1      | main.authorized.func1({0x1397ff8, 0xc000ed81c0}, 0xc00017c900)
gui-1      |    /go/src/labca/main.go:3278 +0x337
gui-1      | net/http.HandlerFunc.ServeHTTP(0xc00017c500?, {0x1397ff8?, 0xc000ed81c0?}, 0x1?)
gui-1      |    /usr/local/go/src/net/http/server.go:2136 +0x29
gui-1      | github.com/gorilla/mux.(*Router).ServeHTTP(0xc0000006c0, {0x1397ff8, 0xc000ed81c0}, 0xc00017c100)
gui-1      |    /root/go/pkg/mod/github.com/gorilla/mux@v1.8.0/mux.go:210 +0x1c5
gui-1      | net/http.serverHandler.ServeHTTP({0x1395930?}, {0x1397ff8?, 0xc000ed81c0?}, 0x6?)
gui-1      |    /usr/local/go/src/net/http/server.go:2938 +0x8e
gui-1      | net/http.(*conn).serve(0xc0010666c0, {0x139ac98, 0xc0001b3f80})
gui-1      |    /usr/local/go/src/net/http/server.go:2009 +0x5f4
gui-1      | created by net/http.(*Server).Serve in goroutine 1
gui-1      |    /usr/local/go/src/net/http/server.go:3086 +0x5cb
```

**Comment by hakwerk** [comment]
I have fixed the issue, it will be part of the next release.

**Comment by hakwerk** [comment]
This was fixed in release [v24.02](https://github.com/hakwerk/labca/releases/tag/v24.02)
